test/test.js

```diff
@@ -0,0 +1,12 @@
+var expect  = require('chai').expect;
+var request = require('request');
+
+describe('Test Server Response', function() {
+    describe ('Server Response', function() {
```

Comment:
Please use 2 spaces as default, i think that you maybe,  don't linted the code. Thats one of the rules at our configuration,.